api/room/schema_query.py

```diff
@@ -280,7 +280,8 @@ def resolve_all_rooms(self, info, **kwargs):
     def resolve_all_remote_rooms(self, info, return_all=None):
         page_token = None
         filter = map_remote_room_location_to_filter()
-        location = 'all' if return_all else get_user_from_db().location
```

Comment:
now that you have deleted the line to return all rooms if return all is true, what if an admin wants to see all remote rooms and his location is in the map filter? ( It will only give him rooms his location )
which means that the only way to get all remote rooms now is when your location is not in the map filter.
I think you should add it again.

Reply:
Thank you @mifeille I've fixed that issue, and let me push some new changes.